### PR TITLE
:memo: Updated README and OpenAPI yml according to the latest changes

### PR DESCRIPTION
**Pull request by izzyblues**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,28 +59,113 @@ Options:
...
+    "id": "http://www.ft.com/thing/d7de27f8-1633-3fcc-b308-c95a2ad7d1cd",
+    "apiUrl": "http://api.ft.com/things/d7de27f8-1633-3fcc-b308-c95a2ad7d1cd",
+    "type": "http://www.ft.com/ontology/Topic",
+    "prefLabel": "Global economic growth"
+  }
+]
 ```
 
-Currently, this endpoint is a proxy to the annotations available in UPP, so it returns a payload consistent to the UPP Public Annotations API.
+### PUT - Writing draft annotations from PAC
```
Comment (khatton-ft) [inline README.md]: Writing ... _to_ PAC

File: README.md [hunk 2]
```diff
@@ -56,28 +59,113 @@ Options:
...
+
+Using curl:
+```
+curl -X PUT \
+  http://localhost:8080/drafts/content/{content-uuid}/annotations \
+  -d '[
+        {
+          "predicate": "http://www.ft.com/ontology/annotation/hasContributor",
+          "id": "http://www.ft.com/thing/5bd49568-6d7c-3c10-a5b0-2f3fd5974a6b",
+          "apiUrl": "http://api.ft.com/people/5bd49568-6d7c-3c10-a5b0-2f3fd5974a6b",
```
Comment (khatton-ft) [inline README.md]: Only the `predicate` and `id` fields are required.

File: README.md (hunk 1 above)
Comment (izzyblues) [reply README.md]: Fixed

File: README.md (hunk 2 above)
Comment (izzyblues) [reply README.md]: Clarified it the paragraph below to this snippet

**Pull request merged**